dot/types/parachain_test.go

```diff
@@ -0,0 +1,271 @@
d6220dd68bba2bbc6a3756154fa5cf878797dacda5fe063c16fd8436a46bc160c9820134984676bc4e14204671db0b5494513b5a7fe4df5d0ea250e554a2080e31802525806814554d3a5677af77a3673b500d3539f04afdcfd720573b608ab0b09939c11cbff67c66f806f5221a61fdcfab5c4fa11c0d7ca12988135efa54ed1d69cd74ceee75480f46a7b15b926904d12cdea281aa28e908ec00a9d07cf8a82bcf585ae95386cc747fdf003d5802d9fe7484687bbbad0448af45553ff3b58903174881b8182392f9757efb143ba61d046a0208302a7a7b3df8041059ca538fef169b8d0012fd33199e57825afe600bbfff192f2b9e0f6b72f5da7e8f1f3147e859987cc129b23e32b2278bada2bc2795d7cd6034e3afa8e9a333d01b36b4e2250e583fe20c06617572612083635c08000000000466726f6e88014920420a08c71a4b56953bf3efbae9a45e737f4cea978bdbc470de907d64e1550005617572610101b491cb7862f8d14fb0e2f6cb8d1287b83a55ee7feb468577ca78d9c120fabe5f6b909448827d93de5eb8bd7afe6c9aea63fef46254208969cdab95fe6d02f18b170000000c000000000808000026662318f64c51f6ed779aa7e32b1b7489bbdb6118812d1b0a2902b7405bc432c663d22aea82481985ee77a58f1e54bb8c64fe22c48df15aa43f69aad1c7276b53c400f7fb9008e65a12443807e49224efa495a9cfa03e6ce5298a88fc744478f3820cb625cff327e14fe2006713cce73555c4336a7d861715aa2429a7d349102f0b4f9017440cf5a06a286856c5795b974f5006aa07357854c3eb4c3892b7f5125cb08a3e444ff92b8868bb8d6d64947c84862353870b0e5121b002929c136db31e000378c0f3241a5408a49bc458bd75af5c0d16b35661e212a981ecf5be8d61707c8adc571e83812efa6aa3673e4423c1b56780c437c500583abf6839a68d096600b21370ef6b990d6922acbfc71de68d751ee2d62fbfc49125bd9709ffe33bbce32c9e23b266d24cc175dcc0d99e7c1182b6ce8824d724e9c34ad852a611e9021145925a3f018c9aaba6b3981aaa8e1e24f0c7d9b38f65643a16d3f55d77ff6ca60a4800e3f4f2125d650231ba3ecaa0a51aeda71beada9dad70479d4571b5f2ebfa480c16116166d9e35a66dcadb999da853bfcd1872563f5d0a785507bf67796e568ec0806617572612083635c080000000005617572610101de7241c718f67134d32b6b5656f8e8f3f31e8d29af28043a37c2b49429a082633c8c7abbe502084c28c4a548ac75f97732521c7e66ea9917e704e5469572558c1e00000013000000003808000026662318f64c51f6ed779aa7e32b1b7489bbdb6118812d1b0a2902b7405bc43278b266c6c3ba60a6e648093bcd62419d4ea68eec772f22b7dfc78c6b255dac57bd149cd2b1b166d73be2ed9409a0bebe842c1901d65b1fd3c5f8d98d82ea2a5c3ae10acc8770276d737e6334c59e468be878e6264c050f48ac3e4a6524b76b37c399c4a820e1a50b2bcbc805192c2609e8372e8ddd8432127f6200427dcb2d72386429023e01fcf193d96cc55f1d4d85db5b2f0a9e3a5fb6855c0e07b891d6581ea371a39cb30eaa0ea2d0b9bbe5734e8793d4db7c366277619bdff9611dfb8d4bba89ec371746a5e689ece6f2a117d2b4e7894776b7a96209e7177446c076874bde474bbe3f7bb78669eb48648bf238110f690c144c097988314989bf9b3d4ef410cd043b32cf71a25a00655ab6a0558e383e546c3728412be664273b51908f4903de6835c05f9da3c6ca7fc65846438da7947f93a152f77bb75a85e3d05385feb47e071300b797adcbfd7d3a94ba4b2e0ab454bfded856f759d2b674839d4adcacb72d2c1361edc442862859f46d384db20db351390b2d261be35211db5232e585f985291a08066e6d6273804852f3c0a603d7da194fbff174c24700f3f89d86b462760033bdcb1663c76c60056e6d62730101e2f5f706646cbfc613a98564ff9b4a958acead02fa488e1f3bbcee114d13594d75e9b79d5f5be257e9da81d3ab97cefe36a620a2384f6f1f8719ec2b23532684280000001d000000" //nolint:all
```

Comment:
Create a separate file `test_data.go` and put all such test data there.

Reply:
@timwu20 @EclesioMeloJunior opinions, yay or nay?

Reply:
It just looks large in the Github's PR interface. In reality, it is just [one line](https://github.com/ChainSafe/gossamer/blob/feat/k/parachain-runtime/lib/parachain/types_test.go#L22) and doesn't affect the readability

Reply:
It looks like there is quite a lot of result data no? Specifically in `TestSessionInfo`